filters/gender_bias/filter.py

```diff
@@ -0,0 +1,348 @@
+import re
+import json
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class GenderBiasFilter(SentenceOperation):
+    tasks = [TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en", "fr", "pl", "ru"]
+
+    def __init__(self, language, feminine_input=[], masculine_input=[]):
+        super().__init__()
+        self.language = language
+        self.feminine_input = feminine_input
+        self.masculine_input = masculine_input
+
+    @staticmethod
+    def flag_sentences(
+        sentences, language, feminine_input=[], masculine_input=[]
+    ):
+        """
+        flag sentences as belonging to the feminine, masculine or neutral groups
```

Comment:
Please add docstrings for 'feminine_input' and 'masculine_input' param.
Also, add docstrings for class.